src/create-reminder-signup/lib/db.ts

```diff
@@ -37,6 +41,7 @@ export function writeOneOffSignup(
 			signup.country,
 			signup.reminder_period,
 			signup.reminder_created_at,
+			now.toISOString(),
```

Comment:
By using `now.toISOString()` instead of just `signup.reminder_created_at`, we will still export signups that are created with timestamps in the past

Reply:
postgres is now managing this filed, instead of us manually, but it works just the same for exporting the signups!